services/user-management/src/main/java/com/foodie/user/service/RolePermissionService.java

```diff
@@ -0,0 +1,39 @@
+package com.foodie.user.service;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+import com.foodie.FoodieBaseRepository;
+import com.foodie.FoodieBaseService;
+import com.foodie.user.model.RolePermission;
+import com.foodie.user.repositories.RolePermissionRepository;
+
+@Service
+public class RolePermissionService implements FoodieBaseService<RolePermission, UUID> {
+
+    private final RolePermissionRepository rolePermissionRepository;
+
+    public RolePermissionService(RolePermissionRepository rolePermissionRepository) {
+        this.rolePermissionRepository = rolePermissionRepository;
+    }
+
+    @Override
+    public FoodieBaseRepository<RolePermission, UUID> getRepository() {
```

Comment:
Why does this exist?

Reply:
Due to my implementation

Reply:
I don't think it's good practice. Can you read more about it please? Repositories should not be exposed this way